content/swiftui/concepts/views/terms/toggle/toggle.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Toggle'
+Description: 'A switch that can be turned on or off.'
```

Comment:
```suggestion
Description: 'Chooses between an "on or off" state based on a Bool type value.'
```

For term entries like this one, the meta-`Description` should begin with a present-tense verb. 😄